### ISSUE TITLE
explanations for fields "Max" And "Min"

### ISSUE THREAD

**Issue by TheRogue88L** [sorare/api#158]
hi guys, thx for your job,
im going to attach an a picture with max and min fields but i dont know what they are:
![image](https://user-images.githubusercontent.com/101823702/158858399-b2b0c4d1-8167-4fd1-aa7f-8d8d53c5d91e.png)
the price of the card is completly different...
do you have an idea about it?

**Comment by jamb0ree** [comment]
same issue.
anyone has an explanation for the `priceRange`  field ?
for example in this query : 
```
query{
  player(slug: "daniil-utkin"){
    cards(rarities:[limited]){
      nodes{
        serialNumber
        priceRange{
          min
          max
        }
      }
    }
  }
}
```

**Comment by nachoroncero** [comment]
Yes, it is because it is measured in Wei, the smallest denomination of Ether. One ether = 10^18 wei

**Comment by jamb0ree** [comment]
> Yes, it is because it is measured in Wei, the smallest denomination of Ether. One ether = 10^18 wei

I get that prices are in wei.
Bur prices that are returned don't have any relation with the actual prices.
For example, from the previous query, the prices returned are : 
```
"priceRange": {
              "min": "36278951836464848",
              "max": "362789518364648512"
            }
```

So prices are : min 0.036278951836464848 eth and max 0.362789518364648512 eth.
Those prices are not in line with actual prices (cards on sale are in the range 0.095-0.109 eth in limited, and in all history of the limited cards, the max price is below 0.16eth).

I can't see what these prices mean.

**Comment by J4F8** [comment]
Hello @redox & all, can you help for this open question on the exact meaning of priceRange{min max} ?

Also, it is surprising that max has exact same digits as min, except one

              "priceRange": {
                "min": "6855031737193763",
                "max": "68550317371937638"
              },

example on player slug diego-jose-palacios-espinoza

@TheRogue88L
